packages/replay/src/bin.ts

```diff
@@ -64,6 +64,11 @@ commandWithGlobalOptions("upload <id>")
 commandWithGlobalOptions("launch [url]")
   .description("Launch the replay browser")
   .option("-b, --browser <browser>", "Browser to launch", "chromium")
+  .option(
+    "--attach <true|false>",
+    "Whether to attach to the browser process after launching",
+    false
```

Comment:
I think a default-false option is right but you're using `detached` in the handler. Let's use `attach` everywhere.